README.md

```diff
@@ -52,11 +52,9 @@ curl http://$DX_IP/tasks
 curl http://$DX_IP/users/ping
 curl http://$DX_IP/users
 curl http://$DX_IP/github/ping
-curl http://$DX_IP/github/members
-curl http://$DX_IP/github/is_member/<github_username>
-curl http://$DX_IP/github/prs?repo=<repo_name>
-curl http://$DX_IP/github/issues?repo=<repo_name>
-curl --globoff "http://$DX_IP/github/issues?repo=<repo_name>&labels=<label 1>&labels=<label 2>"
+curl http://$DX_IP/github/members // must have the proper authorization
+curl http://$DX_IP/github/is_member/<github_username> // check if a paricular GitHub username is part of your GitHub organization
+curl --globoff "http://$DX_IP/github/items?repo=<repo_name>?issue_type=<issues or pull_requests>?labels[]=<label 1>?labels[]=<label 2>?states[]=<state 1>?states[]=<state 2>?limits[]=first?limits[]=100"
```

Comment:
Should the search params be `&` separated?

Reply:
Yes, will fix.